gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/taskexecution/FixTemplate.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.guide.errorprone.taskexecution;
+
+import com.google.common.base.Preconditions;
+
+public final class FixTemplate {
```

Comment:
I'm a little confused - do we need this approach at all? `Project#copy` has two overrides that take two different args:
1. `Action<? super CopySpec>`
2. `Closure` which takes a `CopySpec`

`FileSystemOperations` has a copy method that takes both of these parameters:
1. Direct overload on `Action<? super CopySpec>`
2. Gradle will automatically create a `Closure` version as it's a managed type.

This means we always we back to go from `Project#copy` to `FileSystemOperations#copy`.

Do we need any matchers or a fix template type thing? Can't we just always copy the args of the final method call in the chain? For code like `getProject().getProjectDir()` there are no args so it's fine. For this `copy` case we just copy the original source code as is?

Reply:
> Can't we just always copy the args of the final method call in the chain

That would be sufficient for the copy case! But I thought I'd try and implement something which would support mapping an arbitrary number of args to arbitrary locations along the call chain:

`a(argGroupA).b().c(argGroupC).d()` -> `p(argGroupA).q(argGroupC)`